extras/courses.md

```diff
@@ -8,6 +8,9 @@ or that there was an alternative that fit better into the curriculum.
 
 Courses | Duration | Effort
 :-- | :--: | :--:
+[Learn to Program: The Fundationtals](https://www.coursera.org/learn/learn-to-program) | 7 weeks | 3-4 hours/week
```

Comment:
I don't think we need this, as we already have enough high-quality elementary programming courses.